app/backend/db/chats.ts

```diff
@@ -37,6 +38,32 @@ function insertSelfChat(user: User) {
 	});
 }
 
+async function insertChatWithMembers(userA: User, userB: User) {
+	return db.transaction(async (tx) => {
+		let newChat: { id: string }[] = [];
+		const existingChat = await tx.query.chatTable.findFirst({
+			where: (chat) =>
+				eq(chat.name, `${userA.id}-${userB.id}`) ||
+				eq(chat.name, `${userB.id}-${userA.id}`),
+		});
+		if (!existingChat) {
+			newChat = await tx
+				.insert(chatTable)
+				.values({
+					name: `${userA.id}-${userB.id}`,
+				})
+				.returning({ id: chatTable.id });
```

Comment:
```suggestion
			[{ insertedChatId }] = await tx
				.insert(chatTable)
				.values({
					name: `${userA.id}-${userB.id}`,
				})
				.returning({ insertedChatId: chatTable.id });
```
This eliminates the need for `newChat[0].id` very elegantly right? 😉